### PR TITLE
Add French translations

### PR DESCRIPTION
**Pull request by NathanBnm**
As promised :+1:

### CODE REVIEW

File: po/fr.po [hunk 1]
```diff
@@ -241,111 +274,121 @@ msgid ""
...
 #: src/Widgets/NativeInfoBar.vala:41 src/Widgets/PaidInfoBar.vala:43
 msgid "Dismiss"
-msgstr ""
+msgstr "Masquer"
 
 #. TRANSLATORS: Includes an ellipsis (…) in English to signify the action will be performed in a new window
 #: src/Widgets/NativeInfoBar.vala:46
 msgid "Donate…"
-msgstr ""
+msgstr "Faire un don…""
```
Comment (cassidyjames) [inline po/fr.po]: It looks like this double-quote is breaking the build

File: po/fr.po [hunk 2]
```diff
@@ -66,6 +70,10 @@ msgid ""
 "That means history, cookies, local storage, passwords, etc. are all blown "
 "away as soon as you leave."
 msgstr ""
+"1   Toujours incognito. Dès que vous ouvrez une fenêtre Ephemeral et jusqu'à "
```
Comment (fitojb) [inline po/fr.po]: The original string for this (and the following three translations) contain an emoji number; you should add it to your translation as well

File: po/fr.po [hunk 3]
```diff
@@ -241,111 +274,121 @@ msgid ""
...
-msgstr ""
+msgstr "Ephemeral est une application payante conçue pour elementary OS."
 
 #: src/Widgets/NativeInfoBar.vala:32
 msgid ""
 "Some features may not work properly when running on another OS or desktop "
 "environment."
 msgstr ""
+"Certaines fonctionnalités pourraient ne pas fonctionner correctement sur un autre "
+"système d'exloitation ou environnement de bureau."
```
Comment (fitojb) [inline po/fr.po]: Typo: “exloitation” → “exploitation”

**Comment by NathanBnm** [comment]
Thanks for fixing, I didn't see these errors before commit. I just checked and all seems to be correct now :1:

**Pull request merged**